astropy/extern/_strptime.py

```diff
@@ -249,7 +249,7 @@ def __seqToRE(self, to_convert, directive):
         else:
             return ''
         regex = '|'.join(re_escape(stuff) for stuff in to_convert)
-        regex = '(?P<%s>%s' % (directive, regex)
+        regex = f'(?P<{directive}>{regex}'
```

Comment:
please remove changes to stuff in `extern`

Reply:
Argh, made a small mistake in my grepping that let this through, will fix.